lms/djangoapps/instructor/views/api.py

```diff
@@ -3380,3 +3428,25 @@ def _get_boolean_param(request, param_name):
     values to boolean values.
     """
     return request.POST.get(param_name, False) in ['true', 'True', True]
+
+
+def _use_json(request):
+    """
+    Returns whether the current request expects a response in JSON form.
+    """
```

Comment:
Can you please include in the docstring an explanation of why does this method needs to exist?